lib/google/cloud/gemserver/cli/project.rb

```diff
@@ -70,15 +71,40 @@ def create
           private
 
           ##
-          # Prompts the user to press enter.
+          # @private Checks if the current Google Cloud Platform project
+          # contains a Google App Engine project. If not, one is created.
+          def create_gae_project
+            return if project_exists?
+            puts "Required Google App Engine project does not exist."
+            run_cmd "gcloud app create"
+          end
+
+          ##
+          # @private Runs a given command on the local machine.
+          #
+          # @param [String] args The command to be run.
+          def run_cmd args
+            `#{args}`
```

Comment:
Under what circumstances are you using backticks vs `system()`?

Normally backticks are used to return the STDOUT from the subprocess as a string. If you don't use that output, you might want to prefer `system()` instead.